doc/classes/EditorImportPlugin.xml

```diff
@@ -1,8 +1,54 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorImportPlugin" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Registers a custom resource importer in the editor. Use the class to parse any file and import it as a new resource type.
 	</brief_description>
 	<description>
+		EditorImportPlugins provide a way to extend the editor's resource import functionality. Use them to import resources from custom files or to provide alternatives to the editor's existing importers. Register your [EditorPlugin] with [method EditorPlugin.add_import_plugin].
+
```

Comment:
Please remove the empty lines, as they would be parsed as extraneous paragraphs in the editor and the online docs. Each line of the XML is a paragraph (apart from within codeblocks which work more as preformatted text).